carpics_360.js

```diff
@@ -0,0 +1,540 @@
+                        }
+                    )(thisObj));
+                }
+            })(this));
+
+            $(document).on('touchend', "#" + this.divId, (function(thisObj) {
+                return function(event) {
+                    if (thisObj.mouseDisabled) {
+                        return;
+                    }
+                    thisObj.spinStatus = thisObj.spinDefault;
+                    thisObj.turnStatus = false;
+                    thisObj.turning = false;
+                    thisObj.spinnerDiv.style.cursor = "-webkit-grab";
+                    thisObj.spinnerDiv.style.cursor = "grab";
+                    thisObj.spinnerDiv.style.cursor = "-moz-grab";
+                    $("#" + thisObj.divId ).unbind('touchmove', function(event1) {});
+                }
+            })(this));
+
+        }
+
+        this.setDefaultSpinnerStyles = function() {
+            this.spinnerDiv.style.cursor = "grab";
+            this.spinnerDiv.style.cursor = "-webkit-grab";
+            this.spinnerDiv.style.cursor = "-moz-grab";
+            this.spinnerDiv.style.overflow = "hidden";
+        }
+
+        this.CurrentImage = new CarPicsImage(data[0], 0, this.divId, function() {});
+        this.CurrentImage.HTMLElement.style.display = "block";
+        if (config.linear === "true") {
+            this.LinearReference = {
+                nextLeft: false,
+                goLeft: function() {
+                    this.nextLeft = !this.nextLeft;
+                    return this.nextLeft;
+                }
+            }
+        }
+        this.AllImages = new Array(data.length);
+        this.AllImages[0] = this.CurrentImage;
+        this.LoadCursor = this.CurrentImage;
+        if (!this.mouseDisabled) {
+            this.setDefaultSpinnerStyles();
+        }
+        this.spinnerDiv.appendChild(this.CurrentImage.HTMLElement);
+        if (this.spinStatus === "true") {
+            setInterval((function(thisObj) {
+                return function() {
+                    if (!thisObj.spinStatus) return;
+                    thisObj.DisplayNextImage()
+                }
+            })(this), this.autospinSleep);
+        }
+
+        if (config.spinOnLoad == "true") {
+            this.AllLoadedFunctions.push((function(thisObj) {
+                return function() {
+                    thisObj.spinStatus = 1;
+                    thisObj.spinDefault = 1;
+                    setInterval((function(thisObj) {
+                        return function() {
+                            if (!thisObj.spinStatus) return;
+                            thisObj.DisplayNextImage()
+                        }
+                    })(thisObj), thisObj.autospinSleep);
+                }
+            })(this))
+        }
+        for (var i = 0; i < this.numberOfConnections; i++) {
+            this.getNextImage(this, undefined)();
+        }
+
+    }
+
+    this.CarPicsImage = function(source, index, div, callback) {
```

Comment:
Not a function name.  This is idiomatic to browser (Actually non-typescript, in general) JavaScript.  This is declaration of a class.